t/tags_unit.t

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/perl -w
+
+use Modern::Perl '2017';
+use utf8;
+
+use Test::MockModule;
+use Test::More;
+
+use ProductOpener::Tags qw/get_lc_tagid/;
+
+=head1 Some unit tests for Tags.pm module
+
+=cut
+
+=head2 Unit testing get_lc_tagid
+=cut
+
+# a fake remove stopwords
```

Comment:
So this is just so that we don't have to load Tags.pm to run the test, correct? Maybe we could say so in a comment if that's the case.

Reply:
It's also because I won't my test to be predictable. While here it would depend on the taxonomy file I don't have the hand over.
But yes I should explain why :-)